Fix failing collector unit test

```console
$ cd components/collector
$ just test
Resolved 87 packages in 13ms
Checked 84 packages in 12ms
uv run coverage run -m unittest --quiet
======================================================================
FAIL: test_job_lookback_days_on_edge (tests.source_collectors.gitlab.test_job_runs_within_time_period.GitLabJobRunsWithinTimePeriodTest.test_job_lookback_days_on_edge)
Test the lookback_days around edge of date cut off.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/fniessink/.local/share/uv/python/cpython-3.14.3-macos-aarch64-none/lib/python3.14/asyncio/runners.py", line 127, in run
    return self._loop.run_until_complete(task)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^
  File "/Users/fniessink/.local/share/uv/python/cpython-3.14.3-macos-aarch64-none/lib/python3.14/asyncio/base_events.py", line 719, in run_until_complete
    return future.result()
           ~~~~~~~~~~~~~^^
  File "/Users/fniessink/Developer/quality-time/components/collector/tests/source_collectors/gitlab/test_job_runs_within_time_period.py", line 185, in test_job_lookback_days_on_edge
    self.assert_measurement(response, value="2", entities=expected_entities, landing_url=self.LANDING_URL)
    ~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/fniessink/Developer/quality-time/components/collector/tests/source_collectors/source_collector_test_case.py", line 130, in assert_measurement
    self.__assert_measurement_source_attribute(attribute_key, attribute_value, measurement, source_index)
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/fniessink/Developer/quality-time/components/collector/tests/source_collectors/source_collector_test_case.py", line 147, in __assert_measurement_source_attribute
    self.assertEqual(expected_attribute_value, attribute_value, attribute_key)
    ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError: '2' != '1'
- 2
+ 1
 : value

----------------------------------------------------------------------
Ran 696 tests in 5.597s

FAILED (failures=1)
error: Recipe `py-unit-test` failed on line 161 with exit code 1
error: Recipe `test` failed on line 176 with exit code 1
```

--- a/AGENTS.md
+++ b/AGENTS.md
@@ -1,0 +1,11 @@
+# Agents
+
+## Commit messages
+
+Use a short, imperative subject line describing the change. Add a body with more detail if needed. Reference the GitHub issue with `Fixes #NNNN.` or `Closes #NNNN.` in the body (not the subject — GitHub adds the PR number to the subject on merge). Example:
+
+```
+Fix collector unit test failing during DST transitions
+
+Fixes #12868.
+```

--- a/components/collector/tests/source_collectors/gitlab/test_job_runs_within_time_period.py
+++ b/components/collector/tests/source_collectors/gitlab/test_job_runs_within_time_period.py
@@ -2,7 +2,7 @@
 
 from datetime import datetime, timedelta
 
-from dateutil.tz import tzutc
+from dateutil.tz import tzlocal
 
 from .base import GitLabJobsTestCase
 
@@ -27,7 +27,7 @@ class GitLabJobRunsWithinTimePeriodTest(GitLabJobsTestCase):
 
     async def test_job_lookback_days(self):
         """Test that the job lookback_days are verified."""
-        just_now = datetime.now(tz=tzutc())
+        just_now = datetime.now(tz=tzlocal())
         last_week = just_now - timedelta(weeks=1)
 
         self.gitlab_jobs_json.extend(
@@ -70,7 +70,7 @@ class GitLabJobRunsWithinTimePeriodTest(GitLabJobsTestCase):
 
     async def test_jobs_not_deduplicated(self):
         """Test that the job runs are not deduplicated."""
-        just_now = datetime.now(tz=tzutc())
+        just_now = datetime.now(tz=tzlocal())
         yesterday = just_now - timedelta(days=1)
 
         self.gitlab_jobs_json.extend(
@@ -123,7 +123,7 @@ class GitLabJobRunsWithinTimePeriodTest(GitLabJobsTestCase):
 
     async def test_job_lookback_days_on_edge(self):
         """Test the lookback_days around edge of date cut off."""
-        just_now = datetime.now(tz=tzutc())
+        just_now = datetime.now(tz=tzlocal())
         just_before_cutoff = just_now - timedelta(days=int(self.LOOKBACK_DAYS), minutes=-10)
         just_after_cutoff = just_now - timedelta(days=int(self.LOOKBACK_DAYS), minutes=10)
 

